examples/sensors/webhook.yaml

```diff
@@ -5,14 +5,10 @@ metadata:
   labels:
     sensors.argoproj.io/sensor-controller-instanceid: argo-events
 spec:
+  serviceAccountName: argo-events-sa
+  imageVersion: "v0.6"
```

Comment:
why can't we just use `latest` here? this way, every time we bump versions and cut a tag we don't have to change all the examples? as long as we only publish the tags to docker image repo we should be fine..